packages/esl-website/src/common/playground.less

```diff
@@ -0,0 +1,149 @@
+.uip-landing::before {
```

Comment:
It seems to be not a common styles, let's introduce it in bounds of playground directory
eg `playground/landing.less`